packages/salesforcedx-vscode-core/src/sfdxProject/sfdxProjectPath.ts

```diff
@@ -4,10 +4,10 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import * as vscode from 'vscode';
+import { getRootWorkspacePath } from '../util';
 
 export default class SfdxProjectPath {
   public static getPath() {
-    return vscode.workspace!.workspaceFolders![0].uri.fsPath;
+    return getRootWorkspacePath();
```

Comment:
Instead of only doing this update I think we should replace `SfdxProjectPath`'s usage in favor of the `rootWorkspace` utility in order to have only 1 way of getting `vscode.workspace!.workspaceFolders![0].uri.fsPath` in the core extension.

@allileong what do you think ?

Reply:
How do you guys want to handle the change to replace SfdxProjectPath with this new utility?  In this PR or in a separate PR?

Reply:
Lets have that as part of this PR.

Reply:
I would be fine with doing the refactoring in a separate PR. What do you think @lcampos?

Reply:
Oops, sorry, must not have refreshed my page.

Reply:
I've removed SfdxProjectPath, references, and tests.  Here's the specific commit of that work.

https://github.com/forcedotcom/salesforcedx-vscode/pull/1099/commits/c21af85f37cb9a0d6eb7198097a506b2e277639b

Let me know if this is sufficient to merge this PR in.

Reply:
Awesome, thanks for making that change. This pr looks good to me 👍